># python api.py
Traceback (most recent call last):
  File "/data/oapes_ai/project/CodeGen2-main/api.py", line 2, in <module>
    from transformers import AutoTokenizer, AutoModel
  File "/data/oapes_ai/install_soft/anaconda/envs/codegen2/lib/python3.10/site-packages/transformers/__init__.py", line 26, in <module>
    from . import dependency_versions_check
  File "/data/oapes_ai/install_soft/anaconda/envs/codegen2/lib/python3.10/site-packages/transformers/dependency_versions_check.py", line 16, in <module>
    from .utils.versions import require_version, require_version_core
  File "/data/oapes_ai/install_soft/anaconda/envs/codegen2/lib/python3.10/site-packages/transformers/utils/__init__.py", line 18, in <module>
    from huggingface_hub import get_full_repo_name  # for backward compatibility
  File "/data/oapes_ai/install_soft/anaconda/envs/codegen2/lib/python3.10/site-packages/huggingface_hub-0.21.4-py3.8.egg/huggingface_hub/__init__.py", line 379, in __getattr__
    submod = importlib.import_module(submod_path)
  File "/data/oapes_ai/install_soft/anaconda/envs/codegen2/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/data/oapes_ai/install_soft/anaconda/envs/codegen2/lib/python3.10/site-packages/huggingface_hub-0.21.4-py3.8.egg/huggingface_hub/hf_api.py", line 48, in <module>
    from tqdm.contrib.concurrent import thread_map
ModuleNotFoundError: No module named 'tqdm.contrib'
(codegen2)
